csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2203,7 +2389,7 @@ private predicate pathIntoCallable0(
 ) {
   pathIntoArg(mid, i, outercc, call, ap) and
   callable = resolveCall(call, outercc) and
-  parameterCand(callable, any(int j | j <= i and j >= i), mid.getConfiguration())
+  parameterCand(callable, any(int j | j <= i and j >= i), ap, mid.getConfiguration())
```

Comment:
This change makes it into a triangle join instead of a linear join, so it seems risky - is the join-order ok for all languages?

Reply:
I have not observed that this is problematic.